app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerTestActivityTest.kt

```diff
@@ -306,7 +305,7 @@ class NavigationDrawerTestActivityTest {
         .perform(click())
       onView(withId(R.id.home_fragment_placeholder)).check(matches(isCompletelyDisplayed()))
       onView(isRoot()).perform(orientationLandscape())
-      onView(withId(R.id.home_activity_drawer_layout)).check(matches(isOpen()))
+      onView(withId(R.id.home_activity_drawer_layout)).check(matches(isDisplayed()))
```

Comment:
Both `isOpen` and `isDisplayed` are used for different purpose. 

**[`isOpen`](https://developer.android.com/reference/androidx/test/espresso/contrib/DrawerMatchers#isOpen())**
```
Returns a matcher that verifies that the drawer (with gravity START) is open.
``` 

**[`isDisplayed`](https://developer.android.com/reference/androidx/test/espresso/matcher/ViewMatchers#isdisplayed)**
```
Returns a matcher that matches Views that are currently displayed on the screen to the user.
```